public/app/config/collections/museum/detailFields.js

```diff
@@ -0,0 +1,72 @@
+(function () {
+    'use strict';
+
+    define(function () {
+        return [
+            {
+                key: 'type',
+                labelText: 'Item Type'
+            },
+            {
+                key: 'idno',
+                labelText: 'Accession Number'
+            },
+            {
+                key: 'ItemName',
+                labelText: 'Item Name',
+                placeholder: '(unnamed)'
+            },
+            {
+                key: 'Description',
+                labelText: 'Description',
+                display: 'html'
+            },
+            {
+                key: 'MakersMarks',
+                labelText: 'Makers Marks'
+            },
+            {
+                key: 'Dates',
+                labelText: 'Dates'
+            },
+            {
+                key: 'EarliestYear',
+                labelText: 'Earliest Year'
+            },
+            {
+                key: 'LatestYear',
+                labelText: 'Latest Year'
+            },
+            {
+                key: 'HistoricalDetails',
+                labelText: 'Historical Details'
+            },
+            {
+                key: 'BibliographReferences',
+                labelText: 'Bibliographic References'
+            },
+            {
+                key: 'Subjects',
+                labelText: 'Subjects',
+                parse: true,
+                filter: true,
+                display: 'list'
+            },
+            {
+                key: 'Importance',
+                labelText: 'Importance'
+            },
+            {
+                key: 'StatementOfSignificance',
+                labelText: 'Statement of Significance'
+            },
+            {
+                key: 'Classification',
```

Comment:
`Classification` doesn't appear to have come across:

http://192.168.2.163/rwahs/providence/index.php/editor/objects/ObjectEditor/Edit/Screen34/object_id/3723

http://192.168.2.163/rwahs/providence/service.php/simple/museum_detail?id=3723&noCache=1

http://192.168.2.163:8888/museum/detail/3723